src/components/DailyStatistics/TitleStatistic.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FunctionComponent } from "react";
+import { size, fontSize } from "../../common/styles";
+import { View, StyleSheet, TouchableOpacity } from "react-native";
+import { MaterialCommunityIcons } from "@expo/vector-icons";
+import { AppText } from "../Layout/AppText";
+import { format } from "date-fns";
+
+interface TitleStatisticComponent {
+  totalCount: number | null;
```

Comment:
The `null` is to check to default it to `0` if the value is missing ah?

Reply:
yup it will be passed it as `totalCount={totalCount ?? 0}`